portfolio/src/main/webapp/index.html

```diff
@@ -2,18 +2,24 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Alex Najera</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click <a href="/hello">here</a> to see content created by server code.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <h1>Alex Najera</h1> 
+      <img class="image1" src="/images/meface.png" />
+      
+      <p style="color:grey;">Undergraduate at the University of Texas at Dallas</p>
+      <p style="color:grey;">Majoring in Computer Science</p>
+      <p >I've been learning about cs since freshmen year of high school. I took a random class called Computer Science with some friends and have been studying it since. I'm obsessed with tech and love hearing about all the new innovations that are made every year. That is just one of the many reasons I love computer science. I hope to continue learning, developing my cs skills, and one day do my part in advancing tech and help improve people's lives. </p>
+      <p>I'm motivated by my ambition to learn and I also enjoy a challenge. Some of my favorite hobbies are cooking and running for these reasons. Cooking allows me to explore many recipes and techniques, while running is something I can always continue to push hard for improvement. </p>
+      <img class="image2" src="/images/1backgroundSPS.jpg" />
```

Comment:
Similar comment here about the class name, try to find something a little more meaningful.